### DISCUSSION TITLE
What slicer to select for Orca Slicer ?

### DISCUSSION THREAD

**Discussion by sejtam** [FormerLurker/Octolapse#914] [Q&A]
I am using Orcaslicer 1.6.3 (which is based on BambooSlicer, which is based ... on Prusaslicer)

Do I enable automatic slicer config detection?

How outdated isn the v0.4 set of guides?  ( I have [Octolapse 0.4.3])

Quite a few options/settings referenced in there cannot be found
for my printer (anycubic Mega S i3,  with Klipper installed)

Do I need a different printer setting for Klipper?

I cannot find any 'Build Volume and Origin' settings, 
The G90/G91 influences Extruder setting at all in the firmware settings, or rather these all vanish when. the
server updates the printer profile?

**Comment by RemyLebeau** [comment]
Same issue here.  I'd love to keep using optolapse with orca slicer, but I without doing a full manual calibration, it seems out of the question!

**Reply by RemyLebeau** [reply]
I'm using a Sovol SV06 with Marlin f/w
